test/test.tf

```diff
@@ -63,7 +63,7 @@ module "test_nodes_sydney_channels" {
 }
 
 module "test_lb_sydney" {
-  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=master"
+  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=bump-aws-provider"
```

Comment:
this require merge of: https://github.com/aeternity/infrastructure/pull/555 and will ba set back to `master`